packages/commercetools/composables/src/getters/cartGetters.ts

```diff
@@ -72,13 +74,37 @@ export const getCartTotalItems = (cart: Cart): number => {
 
 export const getFormattedPrice = (price: number) => price as any as string;
 
-export const getCoupons = (cart: Cart): AgnosticCoupon[] => {
+export const getCartCoupons = (cart: Cart): AgnosticCoupon[] => {
```

Comment:
I wouldn't change the name of the existing getter (even if we are not using it) because it will result in a breaking change.

Reply:
I think it doesn't really matter here since in the end, the function is still exported as `getCoupons` inside the `cartGetters` object. What changed is only the naming of the function itself. I just thought it might be good to keep it consistend with the remaining function names in the file but its purely optional here and the change could safely be reverted.

```
const cartGetters: CartGetters<Cart, LineItem> = {
  getTotals: getCartTotals,
  getShippingPrice: getCartShippingPrice,
  getItems: getCartItems,
  getItemName: getCartItemName,
  getItemImage: getCartItemImage,
  getItemPrice: getCartItemPrice,
  getItemQty: getCartItemQty,
  getItemAttributes: getCartItemAttributes,
  getItemSku: getCartItemSku,
  getTotalItems: getCartTotalItems,
  getFormattedPrice,
  getCoupons: getCartCoupons, // still getCoupons
  getDiscounts: getCartDiscounts
};
```